server/sheet.go

```diff
@@ -55,13 +59,16 @@ func (s *Server) registerSheetRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Project ID not found: %d", sheetCreate.ProjectID))
 		}
 
-		sheetCreate.CreatorID = c.Get(getPrincipalIDContextKey()).(int)
-
+		sheetCreate.Source = api.SheetFromBytebase
+		sheetCreate.Type = api.SheetForSQL
+		if sheetCreate.Payload == "" {
```

Comment:
We can add this conversation in the store package (store/sheet.go).

Reply:
Done, PTAL